### PR TITLE
Added documentation link to __init__.py file, per TODO comment

### PR DESCRIPTION
**Pull request by j-dominguez9**
#### Context
What is the purpose of this PR? Is it to
- [ ] add a new feature
- [ ] fix a bug
- [ X] update tests and/or documentation
- [ ] other (please add here)

Please link to any issues this PR addresses.

#### Changelog
What are the changes made in this PR?

Added url to documentation, per the TODO comment on recipes/__init__.py file.

#### Test plan
Please make sure to do each of the following if applicable to your PR. (If you're not sure about any one of these just ask and we will happily help.)

- [ ] run pre-commit hooks and linters (make sure you've first installed via `pre-commit install`)
- [ ] add unit tests for any new functionality
- [ ] update docstrings for any new or updated methods or classes
- [ ] run unit tests via `pytest tests`
- [ ] run recipe tests via `pytest tests -m integration_test`
- [ ] manually run any new or modified recipes with sufficient proof of correctness
	- [ ] include relevant commands and any other artifacts in this summary (pastes of loss curves, eval results, etc.)

### CODE REVIEW

File: recipes/__init__.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # Since we don't want the tests to to incorrectly assume that recipes are
 # importable, we have to explicitly raise an error here.
 
-# TODO: Add proper link to pytorch.org/torchtune/... when the docs are live.
+# Documentation: (https://pytorch.org/torchtune/stable/index.html)
```
Comment (RdoubleA) [inline recipes/__init__.py:19]: let's point to main documentation for now, as that has our latest docs.

**Review by RdoubleA — COMMENTED** [review]
Thanks for catching and fixing this! I just had one comment.

File: recipes/__init__.py (hunk 1 above)
Comment (j-dominguez9) [reply recipes/__init__.py]: Of course. All set! Thanks.

**Comment by joecummings** [comment]
Sorry this is a bit of a misleading TODO. The intent was to come back and add a link to the recipes page (https://pytorch.org/torchtune/main/deep_dives/recipe_deepdive.html) from within the error message itself - not the file. Can you update based on this extra context?

**Review by RdoubleA — APPROVED** [review]

**Pull request merged**